docs-gen/content/rule_set/properties.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Property Entry"
+date: 2023-11-07
+weight: 1
```

Comment:
This is for ordering when we generate HTML with Hugo. If we want it last we should have at least 8

Reply:
Oh, got it.  Thanks.  Please feel free to change, btw.

Reply:
@erikbosch I think it should probably be after Branch Entry and at the same level.